cpp/src/slice2cpp/Gen.cpp

```diff
@@ -730,12 +730,12 @@ Slice::Gen::generate(const UnitPtr& p)
     }
 
     C << "\n#define ICE_BUILDING_GENERATED_CODE";
-    C << "\n#include <";
+    C << "\n#include \"";
```

Comment:
How would this change affect to the users build system?

Reply:
It allows the user to remove a pretty lame `-I.`.